db/model/Gdoc/enrichedToMarkdown.ts

```diff
@@ -62,6 +76,20 @@ function markdownComponent(
     else return undefined
 }
 
+/**
+ * Strips out <Image />, <Video />, etc. components.
+ * Helpful if trying to get a plaintext version of the content because mdast-util-from-markdown
+ * doesn't support these components.
+ */
+export function stripCustomMarkdownComponents(content: string): string {
+    let strippedContent = content
+    for (const componentName of Object.values(CUSTOM_MARKDOWN_COMPONENTS)) {
+        const regex = new RegExp(`<${componentName}.*?/>`, "g")
```

Comment:
Should we make this one a bit more restrictive?

```suggestion
        const regex = new RegExp(`<${componentName}[^\n]*?/>`, "g")
```